.gitignore

```diff
@@ -81,6 +81,3 @@ migration/sqlbindata_test.go
 configuration/confbindata.go
 wit/witservice
 account/tenant
-
-# Ignore artifacts directory
-bin/
```

Comment:
this should stay in the gitignore right? because the compiled artifacts are places here.

Reply:
Sure. I'm just removing duplication - https://github.com/fabric8-services/fabric8-auth/blob/master/.gitignore#L68